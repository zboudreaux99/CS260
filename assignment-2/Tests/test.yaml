test:
  timelimit: 10
  launch: ./tally
  params: '@T/albums.csv @T/requests.csv'
  stderr: false
dueDate: 12/06/2024
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["playlist.h", "playlist.cpp"]
  instructorFiles:
    exclude: ["playlist.h", "playlist.cpp"]
