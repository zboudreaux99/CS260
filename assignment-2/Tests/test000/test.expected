Read 83 songs.
Processed 3 playlist items.

The Great Gig In The Sky Pink Floyd          Dark Side Of The Moon    4 04:49
New Song                 The Who             Who Are You              1 04:17
Had Enough               The Who             Who Are You              2 04:30
905                      The Who             Who Are You              3 03:56
Sister Disco             The Who             Who Are You              4 04:24
Music Must Change        The Who             Who Are You              5 04:40
Trick of the Light       The Who             Who Are You              6 04:14
Guitar and Pen           The Who             Who Are You              7 05:50
Love is Coming Down      The Who             Who Are You              8 04:02
Who Are You              The Who             Who Are You              9 06:24
Lucy In The Sky With Dia The Beatles         Sgt. Peppers Lonely Hear 3 03:26
Total: 50:32

